userdocs/profiles.dev/docs/tutorial-basics/install-a-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 3
+---
+
+# Install a profile
+
+:::info
+
+This stage of the tutorial assumes you have prepared your environment correctly.
+
+Please refer back to the [set up docs](/docs/tutorial-basics/setup) if not.
+:::
+
+To install a profile, we use `pctl`.
+
+First, take care to ensure that you are in your GitOps repo directory on your local
+filesystem. Your GitOps repo is the one you synced to Flux in your cluster in the
+[environment setup](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux) section of this tutorial.
+
+```bash
+git clone <URL to your cluster gitops repo> # if this does not exist locally yet
+cd <path to repository dir>
+```
+
+Once in the correct directory, we can run `pctl` to install a profile.
+_(A breakdown of each flag is provided below.)_
+
+With the following command `pctl` will generate a set of manifests for each profile artifact,
+commit those manifests to a branch in your GitOps repo, push that branch and open a
+PR on your repo to merge the changes.
+
+```bash
+pctl install \
+  --profile-url <URL of profile to install> \
+  --profile-path=. \
+  --create-pr \
+  --pr-repo <gitops repo username or orgname>/<gitops repo name> \
+  --pr-branch install-simple-profile
+```
+
+Above we use the following flags:
+- `--profile-url`. This is the full git URL of the profile you wish to install on your cluster.
+  If you completed the previous section and wrote your own profile, you can use that here.
+  If you chose not to, you can use the following URL to a simple example profile which
+  will install an NGINX server: https://github.com/weaveworks/nginx-profile
+- `--profile-path`. This is the relative path within the profile definition repo which contains the
+  `profile.yaml`. Upstream profile repos can contain multiple profiles separated into
+  different subdirectories. In our case, whether you are using the example repo or the
+  one you created in the previous section, there is just one profile, and the `profile.yaml`
+  is located at the top level: `.`.
+- `--create-pr`. This directs `pctl` to open a PR against the main branch of your GitOps repo.
```

Comment:
I know we said this already, but I think it's good to reiterate that it's only for Github.